src/pages/commitqueue/CommitQueueCard.tsx

```diff
@@ -41,16 +41,12 @@ export const CommitQueueCard: React.FC<Props> = ({
     RemovePatchFromCommitQueueMutation,
     RemovePatchFromCommitQueueMutationVariables
   >(REMOVE_PATCH_FROM_COMMIT_QUEUE);
-  const handleEnroll = async (e): Promise<void> => {
+  const handleEnroll = (e) => {
     e.preventDefault();
-    try {
-      await removePatchFromCommitQueue({
-        variables: { patchId, commitQueueId },
-        refetchQueries: ["CommitQueue"],
-      });
-    } catch (err) {
-      // console.log(err); // TODO: Replace this with better error handling
-    }
+    removePatchFromCommitQueue({
+      variables: { patchId, commitQueueId },
+      refetchQueries: ["CommitQueue"],
+    });
```

Comment:
if a mutation errors and the promise is not caught it crashes the app; i just tested it by trying to configure a patch locally, which always errors. the app crashed. all mutations need to be wrapped in a try/catch